### PR TITLE
Fix automation to fetch

### PR DESCRIPTION
**Pull request by basoundr**
There are 2 changes here
1. Move some of the helpers from test_util to tools_util since the helpers are not depended on the helper binary and can work without any dependencies
2. Introduce TriggerAutomation.bat, which triggers the automation process.

The reason we need a bat file to trigger this and not a csx or a new csx is because running the csx requires the helper binary. Hence we need to run the csx files from the binaries folder but we cannot have binaries in place if we dont fetch them in the first place. This is a cyclic problem that has to be broken with the help of one initial Trigger Batch file

Tagging @KevinH-MS @rchande @TyOverby for review

### CODE REVIEW

File: src/Test/Perf/infra/fetch_build.csx [hunk 1]
```diff
@@ -16,7 +15,7 @@ var destinationFolder = Args.Count() == 2 ? Args[1] : @"C:\Roslyn\Binaries\Relea
 var sourceFolder = $@"\\cpvsbuild\drops\Roslyn\Roslyn-{branch}-Signed-Release";
 
 string latestBuild = null;
-foreach (var folder in Directory.GetFiles(sourceFolder, "????????.?").Reverse())
+foreach (var folder in Directory.GetDirectories(sourceFolder, "????????.?").Reverse())
```
Comment (KevinH-MS) [inline src/Test/Perf/infra/fetch_build.csx]: Consider ```????????.*```

**Comment by KevinH-MS** [comment]
👍

**Comment by rchande** [comment]
:+1:

File: src/Test/Perf/infra/fetch_build.csx (hunk 1 above)
Comment (basoundr) [inline src/Test/Perf/infra/fetch_build.csx]: will modify in the next coming PRs

**Pull request merged**